Update DEP 7 terminology to match current landscape (#91)
django/deps: final/0007-official-projects.rst

    manner.

Shepherd

>     The **Shepherd** is someone with a long history of contributing to Django,

Reviewer: Can we be precise? Perhaps:

```suggestion
    The **Shepherd** is someone who has contributed to Django for at least two years,
```

Author: Yup, I like it.

there is no requirement to wait until all discussions have "finished" before
moving on (as this may take a very long time); instead, they should move
on when they are confident that all viewpoints have been heard and collated.
The Steering Council may refuse the adoption if they think the project was moved
onto the next phase too quickly.

Review & Resolution
-------------------

Once a project has been discussed and the discussion collated by the
Maintenance Team and the Shepherd, it is moved onto review and decision by
the Steering Council. The Shepherd will submit the project, the list of people
signed up for the Maintenance Team, and the collated arguments to the
Steering Council for decision.

The Steering Council are the final authority for deciding on adopting a project
or not. They may choose to rule on the project as a team, or they

> may designate one or more board members to review and decide.

Reviewer 2: I assume the board being referred to here was the Technical Board.

```suggestion
may designate one or more steering council members to review and decide.
```
